Server/MirObjects/SpellObject.cs

```diff
@@ -140,7 +140,7 @@ public void ProcessSpell(MapObject ob)
                         if (ob.Dead) return;
 
                         if (!ob.IsAttackTarget(Caster)) return;
-                        ob.Attacked(((PlayerObject)Caster), Value, DefenceType.MAC, false);
+                        ob.Attacked(((PlayerObject)Caster), (Caster.Stats[Stat.MinSC] + Caster.Stats[Stat.MaxSC]) / 2, DefenceType.MAC, false);
```

Comment:
Sorry. Just one final change needed.

This should still use "Value" - as you originally set Value in the PoisonCloud method "int damage = (Stats[Stat.MinSC] + Stats[Stat.MaxSC]) / 2;"

bonusDmg will still be separate as its another parameter so will not be applied here.

If you could fix this I can appy the merge.

Reply:
If "Value" is used here, the direct damage of the poison cloud will be boosted by Stat.PoisonAttack, I think it should only have a bonus to the green poison damage.But use "Value" here is ok too.

I confirm that the official poison cloud damage is (MaxSC+MinSC)/2. If you use magic.GetDamage, the damage of the PoisonCloud skill will be too large, I tested it.

Reply:
Ah i see. Yes that makes sense not using "Value" here then for direct damage.

However, using the "magic.GetDamage" formula allows for much greater flexibility of how someone may want to use the skill.

If both direct damage and poison damage are both using "(MaxSC+MinSC)/2" then there is no possibility to change how much the spell will do in the files.

I suggest we leave the direct damage using the main "magic.GetDamage" formula (without the PoisonAttack) and have the poison damage use the formula "(MaxSC+MinSC)/2 + PoisonAttack"

Reply:
Okay, I forgot that "magic.GetDamage" can be modified directly in the database. I think your suggestion is the best way.